src/samplers/krb5kdc/stat.rs

```diff
@@ -0,0 +1,706 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use core::convert::TryFrom;
+use core::str::FromStr;
+
+use rustcommon_metrics::*;
+use serde_derive::{Deserialize, Serialize};
+use strum::ParseError;
+use strum_macros::{EnumIter, EnumString, IntoStaticStr};
+
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Deserialize,
+    EnumIter,
+    EnumString,
+    Eq,
+    IntoStaticStr,
+    PartialEq,
+    Hash,
+    Serialize,
+)]
+#[serde(deny_unknown_fields, try_from = "&str", into = "&str")]
+pub enum Krb5kdcStatistic {
+    #[strum(serialize = "krb5kdc/finish_process_as_req/unknown")]
```

Comment:
Can you add some documentation to `docs/METRICS.md` for these new metrics?

Reply:
Add a section on krb5kdc metrics.